testsuite/python-texturesys/src/test_texture_sys.py

```diff
@@ -4,6 +4,8 @@
 # SPDX-License-Identifier: Apache-2.0
 # https://github.com/AcademySoftwareFoundation/OpenImageIO
 
+from __future__ import annotations
```

Comment:
Just out of curiosity, at which Python version is the annotations stuff included and no longer needs to come from `__future__`?

Reply:
This can be removed once OCIO drops support for python 3.9, and only supports 3.10+

Reply:
OK, that will be a while. I think for 3.1 (~Sept release), we will probably raise our minimum from 3.7 to 3.9. But I think we can't go beyond that. Even at my company, I anticipate needing active 3.9 support through the end of the calendar year to be safe.

Reply:
The longer answer is that there are ways to avoid adding this import, but IMHO they are not worth the effort.   For example, if you commit to using `typing.List[X]` instead of `list[str]`, and using `typing.Union[X, Y]` instead of `X | Y`.  It's (understandably) difficult for people to keep track of [which version introduced which feature](https://docs.google.com/document/d/1tCX4rXHUST1Y_4EKTSvoqPiTtYVtLCdGJKlKvaxD6JM/edit?tab=t.0), so using `from __future__ import annotations` is a catchall that let's us not care so we can use these newer features even in python 3.7.

Reply:
No, I'm not worried about the annotation being there at all. It was mostly a curiosity of when in the future we need to remember that they are no longer needed.